bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/servlets/AdaptiveImageServlet.java

```diff
@@ -231,10 +238,13 @@ protected void doGet(@NotNull SlingHttpServletRequest request, @NotNull SlingHtt
                     transformAndStreamAsset(response, componentProperties, resizeWidth, quality, asset, imageType,
                             imageName);
                 }
+                metrics.markRenditionRendered();
```

Comment:
This metric invocation is inconsistent with `markOriginalRenditionUsed`

Reply:
How is it inconsistent?

Reply:
Perhaps I don't understand exactly what you mean by this. To me, the name and description suggest this will count how many times one of the asset's renditions are used.

What it does, in fact, is to count how many successful images are served by the AIS.

Reply:
This counter counts how often a rendition was actually created. Another outcome (instead of the various reasons to return a 404) is the case of returning a 304 "Not modified" [1]. The basic idea behind this counter is actually to determine how often a rendition was created vs how often a 304 was sent (which can be determined as difference between the RenditionRendered and the invocations counter).

[1] https://github.com/adobe/aem-core-wcm-components/blob/master/bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/servlets/AdaptiveImageServlet.java#L219